src/modules/create-market/components/create-market-form-review/create-market-form-review.jsx

```diff
@@ -5,16 +5,24 @@ import { augur } from 'services/augurjs'
 // import BigNumber from 'bignumber.js'
 
 import { formatEtherEstimate, formatEtherTokensEstimate } from 'utils/format-number'
+import { EXPIRY_SOURCE_GENERIC } from 'modules/create-market/constants/new-market-constraints'
 
 import Styles from 'modules/create-market/components/create-market-form-review/create-market-form-review.styles'
 import StylesForm from 'modules/create-market/components/create-market-form/create-market-form.styles'
 
 export default class CreateMarketReview extends Component {
+  static formatResolutionSource(expirySourceType, expirySource) {
```

Comment:
I’m going to refactor this static class method out.  If we were anticipating this method being called outside of this context, I’d be open to doing things this way, but there is a more terse way to have this render into the component.